src/libs/ajax/DataRepo.ts

```diff
@@ -81,6 +78,60 @@ export type SnapshotBuilderSettings = {
   datasetConceptSets?: PrepackagedConceptSet[];
 };
 
+export interface Criteria {
```

Comment:
Right now we have a mix of API names that match the Java types and some that don't. For example we have `SnapshotBuilderConcept`, which does match, and `Criteria`, which doesn't (it would be `SnapshotBuilderCriteria`, I think). Since these names now clash with the UI types and need to be aliased when imported, maybe they should be changed to use the Java type names. If you agree that this is a good idea, can you make a JIRA ticket for it?

Reply:
I would very much agree here - in general I think the types in `DataRepo.ts` should match the Java type names where possible.